src/components/publishing/related_articles/related_article_figure.tsx

```diff
@@ -16,17 +17,37 @@ interface RelatedArticleFigureProps extends React.HTMLProps<HTMLDivElement> {
   }
 }
 
-const RelatedArticleFigure: React.SFC<RelatedArticleFigureProps> = props => {
-  const { article } = props
-  return (
-    <ArticleFigure href={articleHref(article.slug)}>
-      <ImageTitle>
-        <BlockImage src={crop(article.thumbnail_image, { width: 510, height: 340 })} alt={article.thumbnail_title} />
-        <ArticleTitle>{article.thumbnail_title}</ArticleTitle>
-      </ImageTitle>
-      <Byline article={article} layout="condensed" />
-    </ArticleFigure>
-  )
+@track()
+class RelatedArticleFigure extends React.Component<RelatedArticleFigureProps, void> {
+  constructor(props) {
+    super()
+    this.onClick = this.onClick.bind(this)
+  }
+
+  @track((props, [e]) => ({
+    action: "Clicked article impression",
+    article_id: props.article.id,
+    destination_path: e.currentTarget.attributes.href.value,
+    // TODO: What do we put here? According to analytics there are the valid types
+    // [newsletter signup, toc, artist follow, image set, article callout, social, related article]
```

Comment:
I think we'll want to use a new one here, something like `related articles canvas` since this gets used by RelatedArticlesCanvas, but owen/will may have their own prefs.